src/test/java/domain/AttendanceHistoriesTest.java

```diff
@@ -0,0 +1,396 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import fixture.AttendanceDateTimeFixture;
+import fixture.AttendanceHistoriesFixture;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class AttendanceHistoriesTest {
```

Comment:
JUnit5 에서는 테스트 클래스 / 메서드에 public 을 붙일 필요가 없습니다~
```suggestion
class AttendanceHistoriesTest {
```

Reply:
새로 배워갑니다 👍 

8725b56